content-api-post.js

```diff
@@ -42,6 +42,10 @@ exports.handler = function (event, context, callback) {
 
   const maxSize = 2500;
 
+  // Assign the Account record or Champion Fund Id.
+  // We know at least one of these exist before reaching here
+  SalesforceId = data.accountId ? data.accountId : data.championFundId;
```

Comment:
@tbg-jasonfung variable declarations should use a suitable ES6 keyword. If you're not reassigning it that's probably `const`. If you open this in any good IDE you should get a big red warning about this!

Variable name should start with a lowercase for convention with the rest of the file / normal JS style.

`SalesforceId` doesn't really explain what the ID is used for. I think it's reasonable if a bit messy to vary which ID is the primary object linking to the file, but we should probably call it `salesforcePathId` or similar to hint at why we're setting this ternary up in the first place. It probably also makes more sense to do it only when we're about to use it, e.g. just before declaring `path`, otherwise we're executing code for no reason.